.github/label-commenter-config.yml

```diff
@@ -36,6 +36,9 @@ labels:
       issue:
         body: This issue has been labeled with 'component/ui'. 🧰 Here are docs on [Contributing to Meshery UI](https://docs.meshery.io/project/contributing-ui). 🎨 Here is the [Meshery UI Figma File](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=4%3A0) File.
         action: open
+      pr:
+	body: This PR has been labeled with 'component/ui'. 🧰 Here are docs on [Contributing to Meshery UI](https://docs.meshery.io/project/contributing-ui). 🎨 Here is the [Meshery UI Figma File](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=4%3A0) File.
```

Comment:
Not sure if this is correct... But is the indentation right? I also checked your fork on this too but still doubtful. Could you double check it?

Reply:
@alphaX86: CI workflow is not at the right configuration, now it's automatically configured with the help of the `GitHub Actions` extension of Visual Studio Code.
Trust me, it can't arise any conflicts!